databricks_cli/stack/api.py

```diff
@@ -234,6 +245,58 @@ def _update_job(self, job_settings, job_id):
         click.echo("Updating Job")
         self.jobs_client.reset_job({'job_id': job_id, 'new_settings': job_settings})
 
+    def _deploy_workspace(self, resource_properties, physical_id, overwrite):
```

Comment:
It is okey to have `overwrite` here because the scope is narrowed down to only workspace